### PR TITLE
fix: the effect of deleting a page to the trash

### PR DESCRIPTION
**Pull request by om2137**
The current fix for #4282, where After being deleted, the deleted page closes automatically and returns to the all-page list. 
Toast message doesn't work in all (move to trash) buttons. as you can see in the second video attached.

https://github.com/toeverything/AFFiNE/assets/33827410/b5921536-6da9-4f17-ab1e-eb06b534a8da

As the toast message doesn't work I tried out a push notification. I didn't include it in this PR. but I can include it after confirmation. below is the implementation of it.

https://github.com/toeverything/AFFiNE/assets/33827410/ffb15615-0db0-40d3-bb59-cc80b2a9f146

For the (move to trash) button should we create a new Issue?
@pengx17 can you review?

### CODE REVIEW

File: apps/core/src/components/blocksuite/block-suite-header-title/operation-menu.tsx [hunk 1]
```diff
@@ -78,9 +80,11 @@ export const PageMenu = ({ rename, pageId }: PageMenuProps) => {
   }, [mode, setSetting, t]);
   const handleOnConfirm = useCallback(() => {
     removeToTrash(pageId);
+    jumpToSubPath(workspace.id, WorkspaceSubPath.ALL);
+    //toast does't work
     toast(t['com.affine.toastMessage.movedTrash']());
```
Comment (pengx17) [inline apps/core/src/components/blocksuite/block-suite-header-title/operation-menu.tsx:85]: Can you also change this toast to using `pushNotification`? You can search for it for its usage

Comment (om2137) [reply apps/core/src/components/blocksuite/block-suite-header-title/operation-menu.tsx]: > Can you also change this toast to using `pushNotification`? You can search for it for its usage

Done replaced `toast `to `pushNotification` and added `moveToTrash` message description that was suggested in #4282  to all language packs

**Review by pengx17 — APPROVED** [review]

**Comment by om2137** [comment]
@himself65  Are there any changes required?

**Comment by om2137** [comment]
@himself65 can you review this?

**Review by JimmFly — CHANGES_REQUESTED** [review]
I sincerely appreciate your willingness to create a PR for this issue, but I’m deeply sorry, this feature has already implemented a new design. 

However, regarding whether to adopt the message pop-up using notification, our designer @hwangdev97 can provide feedback on this. Based on the designer’s opinion, we can discuss and modify the PR, or we might have to close it.

I’m truly sorry for this situation, and I genuinely acknowledge the time and effort you’ve devoted to this. I assure you that your contributions are highly valued, and I eagerly anticipate your continued partnership and the impactful work that you’ll produce in the future. Once again, my deepest gratitude for your understanding and unwavering support.

https://github.com/toeverything/AFFiNE/assets/102217452/2d5db66f-9aa8-46b6-9ae0-dabf348be531

**Comment by pengx17** [comment]
Hi @om2137 , please see the above comments by @JimmFly

**Comment by om2137** [comment]
> Hi @om2137 , please see the above comments by @JimmFly

I thought the issue is already fixed and PR should be closed

**Pull request closed without merging**